pynestml/codegeneration/nest_builder.py

```diff
@@ -140,21 +148,24 @@ def build(self) -> None:
 
         # first call cmake with all the arguments
         try:
-            result = subprocess.check_call(cmake_cmd, stderr=subprocess.STDOUT, shell=shell,
+            result = subprocess.check_call(cmake_cmd, stderr=stderr, shell=shell, stdout=stdout,
                                            cwd=str(os.path.join(target_path)))
         except subprocess.CalledProcessError as e:
-            raise GeneratedCodeBuildException('Error occurred during \'cmake\'! More detailed error messages can be found in stdout.')
+            raise GeneratedCodeBuildException(
+                'Error occurred during \'cmake\'! More detailed error messages can be found in stdout.')
 
         # now execute make all
         try:
-            subprocess.check_call(make_all_cmd, stderr=subprocess.STDOUT, shell=shell,
+            subprocess.check_call(make_all_cmd, stderr=stderr, shell=shell, stdout=stdout,
                                   cwd=str(os.path.join(target_path)))
         except subprocess.CalledProcessError as e:
-            raise GeneratedCodeBuildException('Error occurred during \'make all\'! More detailed error messages can be found in stdout.')
+            raise GeneratedCodeBuildException(
+                'Error occurred during \'make all\'! More detailed error messages can be found in stdout.')
 
         # finally execute make install
         try:
-            subprocess.check_call(make_install_cmd, stderr=subprocess.STDOUT, shell=shell,
+            subprocess.check_call(make_install_cmd, stderr=stderr, shell=shell, stdout=stdout,
                                   cwd=str(os.path.join(target_path)))
         except subprocess.CalledProcessError as e:
-            raise GeneratedCodeBuildException('Error occurred during \'make install\'! More detailed error messages can be found in stdout.')
+            raise GeneratedCodeBuildException(
+                'Error occurred during \'make install\'! More detailed error messages can be found in stdout.')
```

Comment:
```suggestion
            msg = "Error occurred during 'make install'! More detailed error messages can be found in stdout."
            raise GeneratedCodeBuildException(msg)
```

Yes, I hate line breaks!